PIL/Image.py

```diff
@@ -1814,6 +1893,7 @@ def transform(x, y, matrix):
         return self.transform((w, h), AFFINE, matrix, resample)
 
     def save(self, fp, format=None, **params):
+        # type: (Union[Text, pathlib.Path, BytesIO], Optional[Text], **Any) -> None
```

Comment:
I think this can also be a StringIO.StringIO in py2, but that's a py2 only type.